documentation/milestones/individual_milestones/db_planning.md

```diff
@@ -0,0 +1,185 @@
+- What is the volume of data we expect to work with? Do certain types of databases offer performance optimizations tailored to this volume and type of data (e.g. sharding, indexing, micropartitioning)?
+- Will we need multiple database types for different parts of our data or application?
+
+### Database Management System
+
+Evaluate and select a DBMS for the type of database we've chosen.
+
+Some potential options include: (note some of these DBMS blend multiple paradigms)
+
+- PostgreSQL or MySQL for relational
+- MongoDB or DynamoDB for document-oriented
+- Appache Cassandra or HBase for wide-column
+- Redis or BerkleyDB for key-value
+- Others recommended by dev team
+
+Some factors to consider are:
+
+- What hosting options are available for this DBMS?
+- Can we easily mock or run this DBMS for local development and testing?
+- Are there trusted libraries that support interfacing with this database in the language we've chosen for our API?
+
+### Database Hosting Service
+
+Evaluate and select a service to host and manage our database.
+
+Some potential options include:
+
+- Self hosted on an EC2 instance
+- Managed service from cloud provider (e.g. Amazon RDS)
+- Separate Database as a Service (e.g. Supabase)
+
+Some factors to consider are:
+
+- Is this deployment option FedRAMP compliant?
+- How commonly used is this deployment option? Is it easy for new developers to learn?
+- How expensive is it to host a database with this option, both in terms of direct operating costs and in terms of team resources?
+- Does the service offer features like automatic backups or read replicas?
+
+### Definition of done
+<!-- Required -->
+
+- [ ] The tools and services chosen have been authorized under the Grants.gov ATO (Authority to Operate)
+- [ ] An ADR has been recorded and published to `main` for each of the following choices:
+  - [ ] Database Type
+  - [ ] Database Management System (DBMS)
+  - [ ] Database Hosting Service
+
+### Proposed metrics for measuring goals/value/definition of done
+<!-- Required -->
+
+N/A for planning milestone
+
+### Destination for live updating metrics
+<!-- Required -->
+
+N/A for planning milestone
+
+## Planning
+
+### Assumptions & dependencies
+<!-- Optional -->
+
+What capabilities / milestones do we expect to be in place at the beginning of work
+on this milestone?
+
+- [x] **Onboard dev team:** The dev team will be evaluating and making many of these key architectural decisions
+
+Are there any notable capabilities / milestones do NOT we expect to be in place at the
+beginning of work on this milestone?
+
+- [x] None
+
+### Open questions
+<!-- Optional -->
+
+- [ ] Will this database serve as an OLTP system, OLAP system, or both? Will this database be used for analytics, and is there a need for a second database to be created with data pipelines?
+- [ ] Is the existing Grants.gov database heavy on read or write transactions?
```

Comment:
Note there are thee main existing Grants.gov databases: 

* Oracle on RDS (used for opportunity data, etc)
* MongoDB (I'm not sure exactly what data is in the document store)
* Flat file storage with application data

Reply:
Thanks! Added these notes and created a follow-up ticket (#247) for the other questions.